cmd/openshift-install/create.go

```diff
@@ -418,6 +432,7 @@ func waitForConsole(ctx context.Context, config *rest.Config, directory string)
 	// no route in a row (to show we're still alive).
 	logDownsample := 15
 	silenceRemaining := logDownsample
+	timer.StartTimer("Console")
```

Comment:
this is (accurately) showing `0s` in the fips CI run. not sure if there are other cases where console may take longer to come up.

Reply:
I have never seen a value other than 0s during my tests. Not sure if it's a concern.